pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -60,9 +60,8 @@ type MonitoringStackSpec struct {
 
 	// Time duration to retain data for. Default is '120h',
 	// and must match the regular expression `[0-9]+(ms|s|m|h|d|w|y)` (milliseconds seconds minutes hours days weeks years).
-	// +kubebuilder:validation:Pattern="^[0-9]+(ms|s|m|h|d|w|y)$"
```

Comment:
I hope this is safe to remove when there's validation at https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/types.go#L385

Reply:
Yeah that looks great, nice improvement!